bibliopixel/remote/control.py

```diff
@@ -104,13 +110,32 @@ def __init__(self, layout, animations,
         self.handlers = {
             'run_animation': self.run_animation,
             'stop_animation': self.stop_animation,
-            'get_config': self.get_config
+            'get_config': self.get_config,
```

Comment:
Since these are growing, perhaps we should automate this a bit:

    HANDLER_NAMES = 'get_config', 'run_animation', 'stop_animation', 'trigger_animation', 
    
    def __init__(self, ...):
        self.handlers = {k: v for k, v in self.__dict__ if k in self.HANDLER_NAMES}